### ISSUE TITLE
Template file with extension .tf and not .tf.tmpl

### ISSUE THREAD

**Issue by akerone** [terraform-google-modules/terraform-google-kubernetes-engine#1388]
### TL;DR

The file [terraform-google-kubernetes-engine](https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/tree/b9287de679592a2adcae4d98dcfee33a001f328b)/[autogen](https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/tree/b9287de679592a2adcae4d98dcfee33a001f328b/autogen)/[main](https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/tree/b9287de679592a2adcae4d98dcfee33a001f328b/autogen/main)/variables_defaults.tf has the `.tf` extension instead of `.tf.tmpl` like the rest of the files in that folder.

### Expected behavior

No errors in a project using this module.

### Observed behavior

When used in VS Code with the vscode-terraform extension, it marks the whole project as in Error, because it's trying to analyze all `*.tf` files. This being a template, contains incorrect syntax for a `*.tf` file.

### Terraform Configuration

```hcl
data "google_client_config" "default" {}

provider "kubernetes" {
  host                   = "https://${module.gke.endpoint}"
  token                  = data.google_client_config.default.access_token
  cluster_ca_certificate = base64decode(module.gke.ca_certificate)
}

module "gke" {
  source                     = "terraform-google-modules/kubernetes-engine/google"
  project_id                 = "<PROJECT ID>"
  name                       = "gke-test-1"
  region                     = "us-central1"
  zones                      = ["us-central1-a", "us-central1-b", "us-central1-f"]
  network                    = "vpc-01"
  subnetwork                 = "us-central1-01"
  ip_range_pods              = "us-central1-01-gke-01-pods"
  ip_range_services          = "us-central1-01-gke-01-services"
  http_load_balancing        = false
  network_policy             = false
  horizontal_pod_autoscaling = true
  filestore_csi_driver       = false

  node_pools = [
    {
      name               = "default-node-pool"
      machine_type       = "e2-medium"
      node_locations     = "us-central1-b,us-central1-c"
      min_count          = 1
      max_count          = 100
      local_ssd_count    = 0
      spot               = false
      disk_size_gb       = 100
      disk_type          = "pd-standard"
      image_type         = "COS_CONTAINERD"
      enable_gcfs        = false
      enable_gvnic       = false
      auto_repair        = true
      auto_upgrade       = true
      service_account    = "project-service-account@<PROJECT ID>.iam.gserviceaccount.com"
      preemptible        = false
      initial_node_count = 80
    },
  ]

  node_pools_oauth_scopes = {
    all = [
      "https://www.googleapis.com/auth/logging.write",
      "https://www.googleapis.com/auth/monitoring",
    ]
  }

  node_pools_labels = {
    all = {}

    default-node-pool = {
      default-node-pool = true
    }
  }

  node_pools_metadata = {
    all = {}

    default-node-pool = {
      node-pool-metadata-custom-value = "my-node-pool"
    }
  }

  node_pools_taints = {
    all = []

    default-node-pool = [
      {
        key    = "default-node-pool"
        value  = true
        effect = "PREFER_NO_SCHEDULE"
      },
    ]
  }

  node_pools_tags = {
    all = []

    default-node-pool = [
      "default-node-pool",
    ]
  }
}
```

### Terraform Version

```sh
v1.2.7
```

**Comment by bharathkkb** [comment]
@akerone Thanks for the report! Happy to review a PR updating this.
